core/src/avm2/amf.rs

```diff
@@ -288,6 +294,15 @@ fn get_or_create_element<'gc>(
 pub fn deserialize_value<'gc>(
     activation: &mut Activation<'_, 'gc>,
     val: &AmfValue,
+) -> Result<Value<'gc>, Error<'gc>> {
+    let mut x = BTreeMap::new();
+    deserialize_value_impl(activation, val, &mut x)
```

Comment:
`deserialize_value_impl` only seems to be called here, can't it be inlined into `deserialize_value`?

Reply:
Good catch, I forgot the change the other recursive calls to `deserialize_value_impl`, which has progressed 17782/17336/10174